modules/lidar_oscillation.py

```diff
@@ -0,0 +1,50 @@
+"""
+Representing a LiDAR Oscillation
+"""
+
+from lidar_detection import LidarDetection
```

Comment:
import the module instead of the class - so instead use `from . import lidar_detection`

Reply:
this is just to maintain consistency as well as to avoid confusion because some data structure files contain multiple classes so it is easier to trace back where each object is from this way.